src/main/java/core/basesyntax/model/User.java

```diff
@@ -8,6 +8,17 @@ public class User {
     private String password;
     private Integer age;
 
+    public User() {
+
+    }
+
+    public User(Long id, String login, String password, Integer age) {
+        this.age = age;
+        this.login = login;
+        this.id = id;
```

Comment:
We don't have to add the id value in the User class ourselves, StorageDaoImpl does it for us